docs/concepts/cluster-api.md

```diff
@@ -0,0 +1,11 @@
+# Relation between Gardener API and Cluster API (SIG Cluster Lifecycle)
+
+In essence, the Cluster API harmonizes how to get to clusters, while Gardener goes one step further and also harmonizes the clusters themselves. The Cluster API delegates the specifics to so-called providers for infrastructures or control planes via specific CR(D)s while Gardener only has one cluster CR(D). Different Cluster API providers, e.g. for AWS, Azure, GCP, etc. give you vastly different Kubernetes clusters. In contrast, Gardener gives you the exact same clusters with the exact same K8s version, operating system, control plane configuration like for API server or kubelet, add-ons like overlay network, HPA/VPA, DNS and certificate controllers, ingress and network policy controllers, control plane monitoring and logging stacks, down to the behavior of update procedures, auto-scaling, self-healing, etc. on all supported infrastructures. These homogenous clusters are an essential goal for Gardener as its main purpose is to simplify operations for teams that need to develop and ship software on Kubernetes clusters on a plethora of infrastructures (a.k.a. multi-cloud).
+
+Incidentally, Gardener influenced the Machine API in the Cluster API with its [Machine Controller Manager](https://github.com/gardener/machine-controller-manager) and was the [first to adopt it](https://github.com/kubernetes-sigs/cluster-api/commit/00b1ead264aea6f88585559056c180771cce3815), see also [joint SIG Cluster Lifecycle KubeCon talk](https://www.youtube.com/watch?v=Mtg8jygK3Hs) where @hardikdr from our Gardener team in India spoke.
+
+That means, we follow the [Cluster API](https://github.com/kubernetes-sigs/cluster-api#cluster-api) with great interest and are active members. It was completely overhauled from `v1alpha1` to `v1alpha2`. But because `v1alpha2` made too many assumptions about the bring-up of masters and was enforcing master machine operations (see [here](https://cluster-api.sigs.k8s.io/user/concepts.html#control-plane): “As of `v1alpha2`, Machine-Based is the only control plane type that Cluster API supports”), services that managed their control planes differently like GKE or Gardener couldn't adopt it (e.g. [Google only supports `v1alpha1`](https://cloud.google.com/anthos/gke/docs/on-prem/concepts/cluster-api)). In 2020 [`v1alpha3`](https://kubernetes.io/blog/2020/04/21/cluster-api-v1alpha3-delivers-new-features-and-an-improved-user-experience/) was introduced and made it possible (again) to integrate managed services like GKE or Gardener. The mapping from the Gardener API to the Cluster API is mostly syntactic.
+
```

Comment:
On the implementation history: the Gardener development did not start tackling the bootstrap problem first. It started with the inception design and made the bold assumption that there is already one Kubernetes work plane to bootstrap from. This accelerated the development of Gardener on the basis of an already existing Kubernetes substrate and skills, i.e. launching, scaling managing all components as pods, which made Gardener quickly ready for production (just like many other applications that kickstarted on Kubernetes). Another way to contextualize the development paths is that while Cluster API needed to tackle the challenge bottom-up, Gardener went top-down. Ultimately, both approaches end up having similar requirements in the lowest execution layer and indeed have crosspollinated each other. And in Kubernetes terms, both approches are in a _reconciliation loop_.

Reply:
Hmm... I don't like prolonging this paragraph with the Cluster APIs bootstrap problem or Gardener's implementation history and introducing a bunch of new terms like said bootstrapping, inception, pod-based control planes (you will have noticed that I deliberately didn't go into this area), substrate, bottom-up or top-down that even confuses me here including the cross-pollination ;-), and finally what you mean behind reconciliation.

In other words, while I understand what you are saying, that's not my text and what I wanted to express here and I rather suggest for you to file another PR instead. I hope, that's OK with you/you won't be mad at me, all good, just a different angle.

Reply:
/lgtm then. But I do think this angle makes the relationship much clearer.